kubernetes-manifests/server.service.yaml

```diff
@@ -10,6 +10,12 @@ metadata:
     cloud.google.com/neg: '{"exposed_ports": {"${test_port}":{"name":"${neg_name}"}}}'
 spec:
   type: ClusterIP
+  % if enable_dualstack:
```

Comment:
We need to set this variable KubernetesBaseRunner, see https://github.com/grpc/psm-interop/blob/fc57159059a70534fef4136a15659eca9e140869/framework/test_app/runners/k8s/k8s_xds_server_runner.py#L219-L226

Reply:
Done